src/components/stopwatch/StopWatchController.h

```diff
@@ -0,0 +1,68 @@
+#pragma once
+
+#include <FreeRTOS.h>
+#include <optional>
+#include <timers.h>
+#include "utility/CircularBuffer.h"
+
+namespace Pinetime {
+  namespace System {
+    class SystemTask;
+  }
+
+  namespace Controllers {
+
+    enum class StopWatchStates { Cleared, Running, Paused };
+
+    struct LapInfo {
+      int number = 0;                // Used to label the lap
```

Comment:
I think it makes no difference here due to alignment, but could you shrink the integer types to the smallest applicable? So for here `uint16_t` should be good
Same goes for other member variables / constants, it can save some memory. I wouldn't get too bothered about this, personally I like to size everything but it only really matters for static variables

Reply:
Done.